pkg/scheduler/api/types.go

```diff
@@ -170,6 +170,11 @@ type ExtenderConfig struct {
 	// so the scheduler should only send minimal information about the eligible nodes
 	// assuming that the extender already cached full details of all nodes in the cluster
 	NodeCacheCapable bool
+	// InterestedResources is a pod filter based on pods' requested resources.
+	// If it's empty, every pod will be sent to extender for filter/prioritize/bind operations if FilterVerb/PrioritizeVerb/BindVerb isn't empty.
+	// Otherwise, only pods that requests for one of the resources will be.
+	// +optional
+	InterestedResources []string
```

Comment:
I'd recommend setting this field as follows:
```go
type InterestedResource struct {
 // Has to pass API validation for a compute resource 
 Name v1.ResourceName
 // Scheduler avoids expecting cluster level resources to be attached to nodes
 IsClusterLevel bool
}

// This field is optional
InterestedResources []InterestedResource `json:"interestedResource, omitempty"` 
```

Reply:
IIUC, the `IsClusterLevel` and the additional flag @bsalamat suggested are the same - they both indicate whether this resource should be skipped in the `PodFitsResources` predicate.

Reply:
>  the IsClusterLevel and the additional flag @bsalamat suggested are the same

While that's true in many cases, it is not always true. A resource may be node-level, but we may want it to be ignored by the scheduler and be sent to the extender. For now, we probably want all cluster resources to be ignored though. So, if `IsClusterLevel` is true, scheduler should ignore, but if it is false, it should check the other flag that I suggested.

Reply:
> A resource may be node-level, but we may want it to be ignored by the scheduler and be sent to the extender

@bsalamat as @vishh suggested,  `IsClusterLevel` can be set to true to meet this requirement. If I'm adding another field for example `IsNodeLevel` as you suggested, scheduler does the same thing for both of them. So I think it is redundant to adding `IsNodeLevel`.

Reply:
I think we can have only `IgnoredByScheduler` for now, which will satisfy the use cases discussed here.